cypress/integration/campaign-info-block.js

```diff
@@ -140,6 +141,20 @@ describe('Campaign Info Block', () => {
     });
   });
 
+  /** @test */
+  it('Visit campaign, as an affiliated user, show scholarship modal without "Apply Now" ', () => {
+    const user = userFactory();
+
+    // Auth user visit a campaign page:
+    cy.authVisitCampaignWithSignup(user, exampleCampaign);
+
+    cy.findByTestId('campaign-info-block-container').within(() => {
+      cy.contains('button', 'View Scholarship Details').click();
+    });
+
+    cy.findByTestId('campaign-banner-affiliated').should('not.exist');
```

Comment:
We just need to update this test per the new name and were golden!

Reply:
Oops @Dinnall we need to update this test to look for the `scholarship-modal-signup-button` since we've renamed it. (The `campaign-banner-affiliated` will never appear because it doesn't exist 😅 ). 

Totally fine for a follow-up PR! 

```suggestion
    cy.findByTestId('scholarship-modal-signup-button').should('not.exist');
```